cmd/antrea-agent/agent.go

```diff
@@ -154,6 +154,7 @@ func run(o *Options) error {
 	enableBridgingMode := enableAntreaIPAM && o.config.EnableBridgingMode
 	l7NetworkPolicyEnabled := features.DefaultFeatureGate.Enabled(features.L7NetworkPolicy)
 	nodeNetworkPolicyEnabled := features.DefaultFeatureGate.Enabled(features.NodeNetworkPolicy)
+	nodeLatencyMonitorEnabled := o.config.FeatureGates[string(features.NodeLatencyMonitor)]
```

Comment:
why is this different from the other ones?

Reply:
Instead of using the default featuregate value, I wanted to use the actual value of the config. I noticed that even if I set NodeLatencyMonitor to true in the deployment, it wouldn't set the boolean correctly. Is this expected behavior until I deploy a NodeLatencyMonitor?

Reply:
> Instead of using the default featuregate value, I wanted to use the actual value of the config

This is what all the surrounding lines are doing, otherwise the code would be badly broken.

> I set NodeLatencyMonitor to true in the deployment, it wouldn't set the boolean correctly. Is this expected behavior until I deploy a NodeLatencyMonitor?

No, that's not the expected behavior but I would be surprised if this was the actual behavior as well.

Reply:
Ok, will update, it seemed to just use a constant's value, so I chose to use the config value isntead.